File: app/src/main/java/io/github/lz233/meizugravity/activity/AboutActivity.java
package io.github.lz233.meizugravity.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.KeyEvent;
import android.widget.ImageView;

import io.github.lz233.meizugravity.R;
import io.github.lz233.meizugravity.utils.QRCodeUtil;

public class AboutActivity extends BaseActivity {
    private ImageView aboutImageView;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        //
        aboutImageView = findViewById(R.id.aboutImageView);
        //
        aboutImageView.setImageBitmap(QRCodeUtil.createQRCodeBitmap("https://github.com/lz233/MeizuGravity",100,100));
    }

    @Override
    public boolean onKeyDown(int keyCode, KeyEvent event) {
        if (keyCode == KeyEvent.KEYCODE_MENU){
            finish();
        }
        return super.onKeyDown(keyCode, event);
    }
}

File: app/src/main/java/moe/lz233/meizugravity/activity/AboutActivity.kt
package moe.lz233.meizugravity.activity

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.view.KeyEvent
import moe.lz233.meizugravity.BuildConfig
import moe.lz233.meizugravity.databinding.ActivityAboutBinding
import moe.lz233.meizugravity.design.activity.BaseActivity
import moe.lz233.meizugravity.design.utils.QRCodeUtil

class AboutActivity : BaseActivity() {

    private val viewBuilding by lazy { ActivityAboutBinding.inflate(layoutInflater) }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(viewBuilding.root)
        viewBuilding.aboutImageView.setImageBitmap(QRCodeUtil.createQRCodeBitmap("https://github.com/lz233/MeizuGravity", 100, 100))
        viewBuilding.versionTextView.text = "${BuildConfig.VERSION_NAME} (${BuildConfig.VERSION_CODE})"
    }

    override fun onKeyDown(keyCode: Int, event: KeyEvent?): Boolean {
        when (keyCode) {
            KeyEvent.KEYCODE_MENU -> finish()
        }
        return super.onKeyDown(keyCode, event)
    }

    companion object {
        fun actionStart(context: Context) = context.startActivity(Intent(context, AboutActivity::class.java))
    }
}